.github/workflows/tests.yml

```diff
@@ -0,0 +1,37 @@
+name: Test
+
+on:
+  pull_request:
+
+jobs:
+  tests:
+    name: Running Tests
+    timeout-minutes: 10
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [16.x, 18.x]
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+
+    - name: Start containers
+      run: docker-compose -f "docker-compose-test.yml" up -d --build
```

Comment:
Consistency improvement: we could pull all our docker commands in into yarn commands so they're consistent and easy to find across multiple actions and usages.

Reply:
@ermish I don't have much Docker experience, so I created #232 that we can hopefully pair on. 🙏🏾